src/views/shared/confirmation-modal/confirmation-modal.styles.ts

```diff
@@ -4,12 +4,15 @@ import { Theme } from "src/styles/theme";
 
```

Comment:
the 480 breakpoint is broken

<img width="480" alt="image" src="https://user-images.githubusercontent.com/15896177/229823670-159f9154-3fcd-4b07-baac-ebd0bf661e31.png">

Reply:
Mmmm what do you mean by broken? Which device where you trying to test? It seems that it has very low height.

Reply:
I mean that I can not scroll down and see the complete modal. I'm using Chrome Desktop and reducing the width to 480.